panel.go

```diff
@@ -266,12 +267,13 @@ type (
 		Values *[]string   `json:"values,omitempty"`
 	}
 	Axis struct {
-		Format  string       `json:"format"`
-		LogBase int          `json:"logBase"`
-		Max     *FloatString `json:"max,omitempty"`
-		Min     *FloatString `json:"min,omitempty"`
-		Show    bool         `json:"show"`
-		Label   string       `json:"label,omitempty"`
+		Format   string       `json:"format"`
+		LogBase  int          `json:"logBase"`
+		Decimals int          `json:"decimals"`
```

Comment:
Should probably have `omitempty` since it can be `null` or just not exist: https://github.com/grafana/grafana/blob/99c1c16a046c018d9bf4383261aac09d1b5177e7/devenv/dev-dashboards/panel-common/auto_decimals.json#L774 unless I am missing something.